packages/categorize/src/categorize/index.jsx

```diff
@@ -136,7 +136,8 @@ export class Categorize extends React.Component {
 
     log('[render] disabled: ', model.disabled);
 
-    const { rowLabels } = model;
+    const { rowLabels, categoriesPerRow } = model;
+    const nbOfRows = categories && Math.ceil(categories.length / categoriesPerRow);
```

Comment:
Sanity checks (if categories are null or undefined, how it this going to behave?)